system/preside-objects/admin/security/security_user_two_factor_login_record.cfc

```diff
@@ -11,7 +11,6 @@
 component extends="preside.system.base.SystemPresideObject" displayName="User 2FA Login record" {
 	property name="security_user" relationship="many-to-one" relatedTo="security_user" required=true uniqueindexes="user_machine|1";
 
-	property name="ip_address"     type="string"   dbtype="varchar" maxLength="50"  required=true uniqueindexes="user_machine|2";
-	property name="user_agent"     type="string"   dbtype="varchar" maxLength="255" required=true uniqueindexes="user_machine|3";
-	property name="logged_in_date" type="datetime" dbtype="datetime"                required=false;
+	property name="cookie"         type="string"   dbtype="varchar"  maxLength="255" required=true uniqueindexes="user_machine|2";
```

Comment:
We cannot add a required field that is part of a unique index into an existing table with data - this will fail migration.

An alternative here would be to just use the login record ID as the token value. Then remove the "user_machine" unique index entirely.